alibi_detect/cd/sklearn/classifier.py

```diff
@@ -134,6 +139,30 @@ def _clone_model(self):
                 model.warm_start = False
                 logger.warning('`retrain_from_scratch=True` sets automatically the parameter `warm_start=False`.')
 
+        # oob checks
+        if self.use_oob:
+            if not isinstance(model, RandomForestClassifier):
+                raise ValueError('OOB supported only for RandomForestClassifier. '
+                                 f'Received a model of type {model.__class__.__name__}')
+
+            if self.use_calibration:
+                self.use_calibration = False
+                logger.warning(
+                    'Calibration cannot be user when `use_oob=True`. Setting `use_calibration=False`.'
+                )
+
+            model.oob_score = True
```

Comment:
This is a nice approach! I was going to implement it with an error raised in this instance, this is much better.